src/theme/index.ts

```diff
@@ -19,17 +19,19 @@ const defaultTheme = (theming: ColorScheme): Theme => {
       : enhancePaletteWithShades(darkPaletteConfig);
 
   return {
-    palette,
-    typography,
-    spacing,
-    elevation,
+    globals: {
+      colors: palette,
```

Comment:
colors will not be a different one for each `theming`

so `colors: enhancePaletteWithShades(lightPaletteConfig)` where `enhancePaletteWithShades(lightPaletteConfig)` could be `enhancePaletteWithShades(paletteConfig)`